src/main/webapp/app/quiz/participate/quiz.component.ts

```diff
@@ -679,6 +724,8 @@ export class QuizComponent implements OnInit, OnDestroy {
                         this.shuffle((question as MultipleChoiceQuestion).answerOptions);
                     } else if (question.type === QuestionType.DRAG_AND_DROP) {
                         this.shuffle((question as DragAndDropQuestion).dragItems);
+                    } else if (question.type === QuestionType.SHORT_ANSWER) {
+                        this.shuffle((question as ShortAnswerQuestion).solutions);
```

Comment:
Is shuffling for short answer questions actually necessary? I think we can remove this line!

Reply:
yeah, you are totally right. this is not needed!